pyface/ui/qt4/workbench/split_tab_widget.py

```diff
@@ -49,9 +49,7 @@ def __init__(self, *args):
 
         self.clear()
 
-        QtCore.QObject.connect(QtGui.QApplication.instance(),
-                QtCore.SIGNAL('focusChanged(QWidget *,QWidget *)'),
-                self._focus_changed)
+        QtGui.QApplication.instance().focusChanged.connect(self._focus_changed)
```

Comment:
Is there any reason to worry about the return value of `instance()` being `None` here? (Just wondering whether `QtCore.QObject.connect` automatically ignores `None`s.)